Update documentation (#20)
onflow/flow-js-testing: docs/examples/basic.md

# Basic usage

> Before using any of the methods you need to `init` the framework, basically telling where you Cadence
> code will live. In example below, we put all the Cadence code in the folder named `cadence` one level above the place
> where your test script is located.

Reviewer: ```suggestion
Before using any of the methods you need to `init` the framework. This method tells the library where your Cadence code is. In example below, we put all the Cadence code in a folder named `cadence` one level above the place where your test script is located.
```
